handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -1457,18 +1457,22 @@ public void run() {
         }
 
         @Override
-        public void run(Runnable runnable) {
+        public void run(final Runnable runnable) {
```

Comment:
In 4.1 you will need the final tho

Reply:
This PR is for 4.1 and the `final` is added, though?

Reply:
This is what happens when you review PRs during your vacation 🤦‍♂️... let's ship it

Reply:
green is red after a few 🍻 norman, it is all right 😄